.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**What exactly are your trying to do? Please describe.**
+Sometimes, it's just a matter of documentation where the right method already exists. Don't hesitate to describe your goal with high level details so that we can better discuss and imagine a generalized solution.
+
+**Describe the solution you'd like**
```

Comment:
> For this example, the **Describe the solution you'd like** should show how this feature should look in lours?
> E.g. I want to be able to do dataset.resize(640, 640)

I suppose it can be either this, "I would be so happy if the interface was like this", or some more solution oriented comment, "here's how I think we can implement this"

I'll try to rephrase these sentences, to be fair, they are the default suggested ones :joy:

As for the example, this is a good idea, but be careful, the resize method will be the most demanded one if we put it as en exemple without providing the actual feature 🤯

Reply:
I filled those like a JIRA ticket, sorry 🤣

We can use another example if you want haha but even for us the feature would be interesting to be honest!

Reply:
I just updated the template, let me know if it's better for you

Reply:
Looks good, thanks! :)